contractcourt/channel_arbitrator.go

```diff
@@ -2716,13 +2728,25 @@ func (c *ChannelArbitrator) updateActiveHTLCs() {
 // Nursery for incubation, and ultimate sweeping.
 //
 // NOTE: This MUST be run as a goroutine.
-func (c *ChannelArbitrator) channelAttendant(bestHeight int32) {
+//
+//nolint:funlen
+func (c *ChannelArbitrator) channelAttendant(bestHeight int32,
+	commitSet *CommitSet) {
 
 	// TODO(roasbeef): tell top chain arb we're done
 	defer func() {
 		c.wg.Done()
 	}()
 
+	err := c.progressStateMachineAfterRestart(bestHeight, commitSet)
+	// In case of an error, we return early but we do not shutdown LND,
+	// because there might be other channels that still can be resolved.
+	if err != nil {
+		log.Errorf("Unable to progress state machine after "+
+			"restart: %v", err)
+		return
+	}
```

Comment:
This is a code change behaviour:

Previously we would not startup if the broadcast of the channel commitment failed, now we are just logging an error but we are still continue the daemon.

Reply:
If we want to restore the behavior, we can use `Criticalf` here.